Algorithm-Smallest-Common-Multiple.md

```diff
@@ -1,14 +1,34 @@
 # Explanation:
-This particular problem can be confusing because most people look for the smallest common multiple of the two number but forget the keyword **range.** This means that if you get `[1,5]` then you have to check for the smallest common multiple for all these numbers [1,2,3,4,5] that is evenly divisible by all of them.
+The smallest common multiple between two numbers is the smallest number that
+both numbers can divide into. This concept can be extended to more than two
+numbers as well.
+
+We can first start with just finding the smallest common multiple between two
+numbers. Naively, you can start writing out multiple of each number until you
+write a multiple that exists from both numbers.
+
+An example would be the numbers `3` and `4`. The multiples of `3` are `3, 6, 9,
+12, 15, 18, ...` and the multiples of `4` are `4, 8, 12, 16, 20, ...`.  The
+first smallest number we run into in both lists is `12` so this is the smallest
+common multiple between `3` and `4`.
+
+This problem can be confusing because most people look for the smallest common
+multiple of just the two numbers but forget the keyword **range**. However, this
+means that if you are given `[1,5]`, then you have to check for the smallest
+common multiple for all the numbers `[1,2,3,4,5]` that is evenly divisible by
+all of them.
 
 ## Hint: 1
-Create an array with all the numbers that are missing from the original array to make it easier to check when having to check for even division.
+Create an array with all the numbers that are missing from the original array to
+make it easier to check when having to check for even division.
 
 ## Hint: 2
-You can use modulo `%` to check if the reminder is 0, which means it is evenly divisible.
+You can use modulo operator (`%`) to check if the reminder is 0, which means it is evenly divisible.

```

Comment:
Technically this is the `remainder` operator - JavaScript does not have a `modulo` operator.  The main difference is what happens with a negative number:
http://www.sitecrafting.com/blog/modulus-remainder/